src/Analysis/GWASResults/Views/Home/HomeTable/ActionsDropdown/ActionsDropdown.test.jsx

```diff
@@ -2,14 +2,29 @@ import React from 'react';
 import { render, fireEvent, waitFor } from '@testing-library/react';
 import '@testing-library/jest-dom';
 import ActionsDropdown from './ActionsDropdown';
+import PHASES from '../../../../Utils/PhasesEnumeration';
 
 describe('ActionsDropdown', () => {
-  it('should open the dropdown menu when the button is clicked', () => {
-    const { getByRole, getByText } = render(<ActionsDropdown />);
+  it('should open the dropdown menu when the button is clicked and Retry option should be disabled for the Phase "running"', () => {
+    const record = {phase: PHASES.Running};
+    const { getByRole, getByText } = render(<ActionsDropdown record={record}/>);
     const dropdownButton = getByRole('button');
     waitFor(() => {
       fireEvent.click(dropdownButton);
     });
     expect(getByText('Download')).toBeInTheDocument();
+    expect(getByText('Retry')).toBeInTheDocument();
+    expect(getByText('Retry').parentElement.parentElement).toHaveClass('ant-dropdown-menu-item-disabled');
```

Comment:
this line seems a bit fragile....but I have not found another working solution yet...